test/automation/src/sql/configurePythonDialog.ts

```diff
@@ -19,28 +19,27 @@ export class ConfigurePythonDialog extends Dialog {
 	}
 
 	async installPython(): Promise<void> {
-		const dialog = '.modal .modal-dialog';
-		await this.code.waitAndClick(dialog);
+		const dialogPageInView = '.modal .modal-body .dialogModal-pane:not(.dialogModal-hidden)';
+		const dialogButtonInView = '.modal .modal-footer .footer-button:not(.dialogModal-hidden)';
 
-		const newPythonInstallation = `${dialog} .modal-body input[aria-label="New Python installation"]`;
+		const newPythonInstallation = `${dialogPageInView} input[aria-label="New Python installation"]`;
 		await this.code.waitAndClick(newPythonInstallation);
 
 		// Wait for the python install location to be loaded before clicking the next button.
 		// There may be a timing issue where the smoke test attempts to go to the next page before
 		// the contents are loaded, causing the test to fail.
-		const pythonInstallLocationDropdownValue = `${dialog} select[aria-label="Python Install Location"] option`;
+		const pythonInstallLocationDropdownValue = `${dialogPageInView} option[value*="/azuredatastudio-python (Default)"]`;
```

Comment:
`*=` looks for a partial match - I'm not using the full path since the path will be different when the test is ran in the pipeline vs locally